web/src/views/DandisetLandingView/ContactDialog.vue

```diff
@@ -0,0 +1,130 @@
+    >
+      <v-card-title class="pb-0" style="min-width: fit-content;">
+        Select an e-mail Recipient!
+      </v-card-title>
+      <v-list>
+        <v-tooltip
+          :disabled="!owners ? false : loggedIn()"
+          open-on-hover
+          right
+        >
+          <template #activator="{ on }">
+            <div
+            v-on="on"
+            >
+              <v-list-item
+                :disabled="!loggedIn() || !owners"
+                :selectable="!loggedIn() || !owners"
+                :href="makeTemplate(owners)"
+              >
+                <v-icon
+                  color="primary"
+                  left
+                  small
+                >
+                  mdi-card-account-mail
+                </v-icon>
+                Dandiset Owners
+              </v-list-item>
+            </div>
+          </template>
+          <span v-if="!loggedIn()"> You must me logged in to contact the owner </span>
+          <span v-if="!owners"> No owner e-mail available </span>
+        </v-tooltip>
+      <v-divider />
+        <v-tooltip
+            :disabled="contacts === undefined || contacts?.length > 0 "
+            open-on-hover
+            right
+          >
+
+          <template #activator="{ on }">
+            <div v-on="on">
+              <v-list-item
+                :disabled=" contacts?.length === 0 || !contacts"
+                :selectable="!contacts"
+                :href="makeTemplate(contacts)"
+              >
+                <v-icon
+                  color="primary"
+                  left
+                  small
+                >
+                  mdi-card-account-mail
+                </v-icon>
+                Contact Person
+              </v-list-item>
+            </div>
+            </template>
+          <span> No contact e-mail available </span>
+        </v-tooltip>
+      </v-list>
+    </v-card>
+  </v-menu>
+</template>
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useDandisetStore } from '@/stores/dandiset';
+import { loggedIn } from '@/rest';
+import type { User, Person, Organization, Email} from '@/types';
+
+const store = useDandisetStore();
+const owners = computed(() => store.owners?.map((owner: User) => owner.email));
+const contacts = computed(() =>
+  store.dandiset?.metadata?.contributor?.filter(
+    (contact: Person | Organization) =>
+      contact.roleName?.includes("dcite:ContactPerson"))
+    .map((contact: Person | Organization) =>
+      contact.email as Email
+    )
+);
+const currentDandiset = computed(() => store.dandiset);
+
+const makeTemplate = (contacts: string[] | undefined) => {
+  if (contacts === undefined) {
+    throw new Error('Contact is undefined.');
+  }
+  if (currentDandiset.value){
+    const subject = encodeURIComponent(`Regarding Dandiset ${currentDandiset.value.name} ${currentDandiset.value.metadata?.id}`);
+    const contact = contacts.join(',');
+    return `mailto:${contact}?subject=${subject}`;
```

Comment:
In the deploy preview for this PR, this is working for the "contact person" option, but not for "dandiset owners". Example: in [Dandiset 200560](https://deploy-preview-1840--gui-staging-dandiarchive-org.netlify.app/dandiset/200560), the mailto link includes a blank email address (it reads `mailto:?subject=[...]`).

Reply:
This is due to the deploy preview going to the production API, which does not have the changes from this PR that are needed to correctly populate the owners list.